integrations/mistral/src/haystack_integrations/components/generators/mistral/chat/chat_generator.py

```diff
@@ -54,10 +60,11 @@ class MistralChatGenerator(OpenAIChatGenerator):
     def __init__(
         self,
         api_key: Secret = Secret.from_env_var("MISTRAL_API_KEY"),
-        model: str = "mistral-tiny",
+        model: str = "mistral-small-latest",
```

Comment:
if there is a good reason to change the default model, let's mark this PR as breaking ("!" in the title).
In any case, the title needs to be aligned with the common format.

Reply:
ok yeah, will do mark it as breaking, couldn't find this model listed any more nor supporting FC, see https://docs.mistral.ai/capabilities/function_calling/